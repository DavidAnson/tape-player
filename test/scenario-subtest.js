// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("outer test", (test) => {
  test.pass();
  test.test("inner test", (inner) => {
    inner.pass();
    inner.fail("inner");
    inner.end();
  });
  test.fail("outer");
  test.end();
});
