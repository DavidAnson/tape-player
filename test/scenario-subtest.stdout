outer test
  scenario-subtest.js:15:8
  Message:  outer
  Operator: fail
inner test
  scenario-subtest.js:12:11
  Message:  inner
  Operator: fail

Tests:    2
Asserts:  4
Failures: 2
